### DISCUSSION TITLE
es/addons/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#489] [Page comments]
# es/addons/

Esta página lista todos los addons que están implementados en Scratch Addons.

https://scratchaddons.com/es/addons/

**Comment by violinista8** [comment]
Could you please add more space to proyects please i need it with urgency, a proyect that i was making for 1 year and a half cant load because there are too much costumes and i need more space please add some to scratch addons please

**Reply by WorldLanguages** [reply]
It's not possible.

**Comment by EndiMakes** [comment]
omg

**Comment by pokemesh101092-netizen** [comment]
The
